src/lightbox/downloadImage.js

```diff
@@ -35,27 +34,35 @@ const androidEnsureStoragePermission = async (): Promise<void> => {
 };
 
 export default async (src: string, auth: Auth): Promise<mixed> => {
```

Comment:
Let's tighten up the return type of this function, now that the returned `Promise` might resolve to the string "browser".

Reply:
The return type is 
```Promise<{|
  url: string,
  openedInBrowser: boolean,
|}>``` now.